### PR TITLE
stage | GIDK-3bj75g | Aditi | env difference fixed

### PR DESCRIPTION
**Pull request by aditi-tiwari17**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
https://app.clickup.com/t/3bj75g

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

File: apps/web-giddh/src/app/gst/filing/tabs/overview/view-transactions/view-transactions.component.ts [hunk 1]
```diff
@@ -29,7 +29,7 @@ export const filterTransaction = {
     // tslint:disable-next-line:component-selector
     selector: 'view-transactions',
     templateUrl: './view-transactions.component.html',
-    styleUrls: ['view-transactions.component.css'],
+    styleUrls: ['view-transactions.component.scss'],
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/gst/filing/tabs/overview/view-transactions/view-transactions.component.ts:32]: Please change to `./view-transactions.component.scss`

File: apps/web-giddh/src/app/gst/filing/tabs/reconcilation/reconcilation.component.ts [hunk 2]
```diff
@@ -11,27 +11,28 @@ import { GstReconcileActions } from '../../../../actions/gst-reconcile/GstReconc
...
-            })),
-            state('out', style({
-                transform: 'translate3d(100%, 0, 0)'
-            })),
-            transition('in <=> out', animate('400ms ease-in-out')),
-        ])
-    ]
+	selector: 'reconcile',
+	templateUrl: './reconcilation.component.html',
+	styleUrls: ['reconcilation.component.scss'],
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/gst/filing/tabs/reconcilation/reconcilation.component.ts:16]: Please change to `./reconcilation.component.scss`

File: apps/web-giddh/src/app/ledger/components/updateLedgerEntryPanel/updateLedgerEntryPanel.component.scss [hunk 3]
```diff
@@ -83,7 +83,7 @@ ng-select .select-drop i.fa.fa-caret-up::before {
     padding: 2px 5px !important;
 }
 
-.no-padding {
+.p-0 {
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/ledger/components/updateLedgerEntryPanel/updateLedgerEntryPanel.component.scss:86]: Please check if this style is now required as we have global style for `padding: 0`

File: apps/web-giddh/src/app/shared/header/header.component.scss [hunk 4]
```diff
@@ -153,7 +153,7 @@ a[vr-item]:focus {
     position: relative;
     font-family: "opensans-semibold";
 }
-.no-padding {
+.p-0 {
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/shared/header/header.component.scss:156]: Please check if this style is now required as we have global style for `padding: 0`

**Review by prateek-giddh — CHANGES_REQUESTED** [review]

**Review by ravinderwalkover — APPROVED** [review]

**Review by prateek-giddh — APPROVED** [review]

**Pull request merged**